docs/pages/components/Combobox.mdx

```diff
@@ -0,0 +1,347 @@
+    </ComboboxOption>
+    <ComboboxOption 
+      description="Revive your taste buds with this undead burst of citrusy hops."
+      value="Zombie Zest Pale Ale"
+    >
+      Zombie Zest Pale Ale
+    </ComboboxOption>
+  </Combobox>
+</FieldWrap>
+```
+
+### Required
+
+```jsx example
+<FieldWrap>
+  <Combobox label="Is this combobox required?" required>
+    <ComboboxOption>Yes</ComboboxOption>
+    <ComboboxOption>No</ComboboxOption>
+    <ComboboxOption>I'm not sure</ComboboxOption>
+  </Combobox>
+</FieldWrap>
+```
+
+### Error
+
+```jsx example
+<FieldWrap>
+  <Combobox 
+    label="Does this combobox have an error?"
+    error="You must select an option."
+    required
+  >
+    <ComboboxOption>Yes</ComboboxOption>
+    <ComboboxOption>No</ComboboxOption>
+    <ComboboxOption>I'm not sure</ComboboxOption>
+  </Combobox>
+</FieldWrap>
+```
+
+### Grouping
+
+```jsx example
+<FieldWrap>
+  <Combobox label="Countries">
+    <ComboboxGroup label="North America">
+      <ComboboxOption>Canada</ComboboxOption>
+      <ComboboxOption>Mexico</ComboboxOption>
+      <ComboboxOption>United States</ComboboxOption>
+    </ComboboxGroup>
+    <ComboboxGroup label="South America">
+      <ComboboxOption>Argentina</ComboboxOption>
+      <ComboboxOption>Bolivia</ComboboxOption>
+      <ComboboxOption>Brazil</ComboboxOption>
+      <ComboboxOption>Chile</ComboboxOption>
+      <ComboboxOption>Columbia</ComboboxOption>
+      <ComboboxOption>Ecuador</ComboboxOption>
+      <ComboboxOption>Falkland Islands</ComboboxOption>
+      <ComboboxOption>French Guiana</ComboboxOption>
+      <ComboboxOption>Guyana</ComboboxOption>
+      <ComboboxOption>Paraguay</ComboboxOption>
+      <ComboboxOption>Peru</ComboboxOption>
+      <ComboboxOption>Suriname</ComboboxOption>
+      <ComboboxOption>Uruguay</ComboboxOption>
+      <ComboboxOption>Venezuela</ComboboxOption>
+    </ComboboxGroup>
+  </Combobox>
+</FieldWrap>
+```
+
+### Uncontrolled
+
+To set the initial value for uncontrolled Comboboxes, use `defaultValue`.
+
+```jsx example
+<FieldWrap>
+  <Combobox label="Fruit" defaultValue="Peach">
+    <ComboboxOption value="Apple">🍎 Apple</ComboboxOption>
+    <ComboboxOption value="Banana">🍌 Banana</ComboboxOption>
+    <ComboboxOption value="Cucumber">🥒 Cucumber</ComboboxOption>
+    <ComboboxOption value="Orange">🍊 Orange</ComboboxOption>
+    <ComboboxOption value="Peach">🍑 Peach</ComboboxOption>
+    <ComboboxOption value="Pear">🍐 Pear</ComboboxOption>
+  </Combobox>
+</FieldWrap>
+```
+
+### Controlled
+
+Comboboxes can also be controlled, by setting the `value` prop and updating the value from event callbacks.
+
+<Note>
+  `onChange` and `onSelectionChange` have different behaviors and special consideration needs need to be made when `Combobox` is controlled. `onChange` gets called when the value of the input field changes from user input or selections while `onSelectionChange` is only called when a selection changes from the listbox.
```

Comment:
The wording of the first sentence is a bit confusing to me. Did you mean "When being used in a uncontrolled `Combobox`,`onChange` and `onSelectionChange` have different behaviors and special consideration needs"

Reply:
Looks like I had an extra word there in there, I removed it - does that help?